sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -114,16 +114,16 @@ public partial class MatchConditions
         public Azure.ETag? IfMatch { get { throw null; } set { } }
         public Azure.ETag? IfNoneMatch { get { throw null; } set { } }
     }
-    public abstract partial class NullableResponse<T>
+    public abstract partial class NullableResponse<T> : System.ClientModel.OptionalOutputMessage<T>
     {
-        protected NullableResponse() { }
-        public abstract bool HasValue { get; }
-        public abstract T? Value { get; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        protected NullableResponse() : base (default(T), default(System.ClientModel.Primitives.PipelineResponse)) { }
+        protected NullableResponse(T? value, Azure.Response response) : base (default(T), default(System.ClientModel.Primitives.PipelineResponse)) { }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override bool Equals(object? obj) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override int GetHashCode() { throw null; }
-        public abstract Azure.Response GetRawResponse();
+        public virtual new Azure.Response GetRawResponse() { throw null; }
```

Comment:
The abstract GetRawResponse member on `NullableResponse<T>` becomes virtual so that we don't need to add a new `Response` field beyond the `OutputMessage` PipelineResponse field in derived types.

Reply:
An alternative is to keep the newslotted GetRawResponse abstract, but then anyone outside Azure.Core who inherits from `Response<T>` would not have access to the PipelineResponse on the base type.